scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -130,6 +132,10 @@ case object RedundantBraces extends Rewrite {
         !isProcedureSyntax(d) &&
         !disqualifiedByUnit
 
+      case p: Term.Function
+          if isBlockFunction(p) && ctx.style.activeFor(Edition(2019, 11)) =>
```

Comment:
as requested, added an `Edition` barrier

Reply:
Let’s bind this to a val to avoid repeated evaluation

Reply:
Done, uploading now.